components/aws/sagemaker/batch_transform/README.md

```diff
@@ -0,0 +1,98 @@
+model_name | The name of the model that you want to use for the transform job. Model name must be the name of an existing Amazon SageMaker model within an AWS Region in an AWS account | No | No | String | | |
+max_concurrent | The maximum number of parallel requests that can be sent to each instance in a transform job | Yes | Yes | Integer | | 0 |
+max_payload | The maximum allowed size of the payload, in MB | Yes | Yes | Integer | The value in max_payload must be greater than, or equal to, the size of a single record | 6 |
+batch_strategy | The number of records to include in a mini-batch for an HTTP inference request | Yes | Yes | String | | |
+environment | The environment variables to set in the Docker container | Yes | Yes | Dict | Maximum length of 1024. Key Pattern: `[a-zA-Z_][a-zA-Z0-9_]*`. Value Pattern: `[\S\s]*`. Upto 16 key and values entries in the map | |
+
+The following parameters construct [TransformInput](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TransformInput.html) object of the input. These describe the input source and the way the transform job consumes it.
+
+Argument        | Description                 | Optional (in pipeline definition) | Optional (in UI) | Data type  | Accepted values | Default    |
+:---            | :----------                 | :----------                       | :----------      | :----------| :----------     | :----------|
+input_location | The S3 location of the data source that is associated with a channel. [Read more on S3Uri](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TransformS3DataSource.html) | No | No | String | | |
+data_type | Used by Sagemaker to identify the objects from the S3 bucket to be used for batch transform. [Read more on S3DataType](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TransformS3DataSource.html) | Yes | Yes | String | `ManifestFile`, `S3Prefix`, `AugmentedManifestFile`| S3Prefix |
+content_type | The multipurpose internet mail extension (MIME) type of the data. Amazon SageMaker uses the MIME type with each http call to transfer data to the transform job | Yes | Yes | String | | |
+split_type | The method to use to split the transform job data files into smaller batches | Yes | Yes | String | `Line`, `RecordIO`, `TFRecord`, `None` | None |
+compression_type | If the transform data is compressed, specify the compression type | Yes | Yes | String | `GZip`, `None` | None |
+
+* input_location and data_type parameters above are used to construct [S3DataSource](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TransformS3DataSource.html) object which is part of [TransformDataSource](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TransformDataSource.html) object in [TransformInput](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TransformInput.html) part of the batch transform API.
```

Comment:
nit: Use `` around parameter names when writing them in paragraphs.

Reply:
done

Reply:
done